aero-sample-apps/debian/copyright

```diff
@@ -0,0 +1,9 @@
+Files: *
+Copyright: 2018 Intel Corporation
+License: GPL-2
```

Comment:
As this is a meta-package, is there anything else besides the debian/* dir?
The license of each individual package is handled by themselves.